client/source/useModal.js

```diff
@@ -0,0 +1,13 @@
+import { useState } from 'react';
+
+const useModal = function useModal() {
+  const [showModal, setShowModal] = useState(false);
+
+  return {
+    isOpen: showModal,
+    onOpen: () => setShowModal(true),
```

Comment:
onOpen: () => { document.body.classList.add('stop-scrolling'); setShowModal(true); },
    onClose: () => { document.body.classList.remove('stop-scrolling'); setShowModal(false); }

Reply:
Okay, I put that in there too

Reply:
Sorry I changed my mind, confusing I know!